dbms/src/Flash/Statistics/ExecutorStatisticsCollector.cpp

```diff
@@ -142,6 +142,15 @@ tipb::SelectResponse ExecutorStatisticsCollector::genExecutionSummaryResponse()
     return response;
 }
 
+tipb::TiFlashExecutionInfo ExecutorStatisticsCollector::genTiFlashExecutionInfo()
+{
+    tipb::SelectResponse response = genExecutionSummaryResponse();
```

Comment:
`genExecutionSummaryResponse` will also generate remote execution summary, which in this case will be all zero, is that expected, or do we need these zero execution summaries?

Reply:
Yes, since remote executions won't use DispatchMPPTask interface, its execution summary needs to be included here now if has.